test/MVCFramework45.FunctionalTests/FunctionalTest/DependencyTelemetryMvcTests.cs

```diff
@@ -11,12 +11,80 @@ namespace SampleWebAppIntegration.FunctionalTest
     using Microsoft.ApplicationInsights.Channel;
     using Microsoft.ApplicationInsights.DataContracts;
     using Xunit;
+    using Microsoft.ApplicationInsights.Extensibility;
+    using Microsoft.ApplicationInsights.DependencyCollector;
+    using Microsoft.Extensions.DependencyInjection;
 
     public class DependencyTelemetryMvcTests : TelemetryTestsBase
     {
         private const string assemblyName = "MVCFramework45.FunctionalTests";
 
         [Fact]
+        public void OperationIdOfOutgoingRequestIsPropagated()
```

Comment:
It makes sense to me at first. But to test the E2E cross component correlation functionality, the DependencyCollector and the listener (this project) are both needed, so it's hard to say it's whose job. Since it's just a test case, I'll leave it here.